test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpVerbatimBlockTest/InnerImplicitExpressionAcceptsTrailingNewlineInDesignTimeMode.syntaxtree.txt

```diff
@@ -0,0 +1,8 @@
+Statement block - NullParentChunkGenerator - 8 - (0:0,0)
```

Comment:
Looks like adding the block length here breaks in crossplat when there is a newline involved. I'm looking in to it.

Reply:
Yeah we need a strategy for that. I suggest that we normalize it one way or the other.